src/results/financial.jl

```diff
@@ -54,19 +71,56 @@ function add_financial_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _
         r["lifecycle_om_costs_after_tax"]
     r["lifecycle_capital_costs"] = value(m[Symbol("TotalTechCapCosts"*_n)] + m[Symbol("TotalStorageCapCosts"*_n)])
     r["initial_capital_costs"] = initial_capex(m, p; _n=_n)
-    r["initial_capital_costs_after_incentives"] = initial_capex_after_incentives(m, p, r["lifecycle_capital_costs"]; _n=_n)
-
     future_replacement_cost, present_replacement_cost = replacement_costs_future_and_present(m, p; _n=_n)
-    r["replacement_costs"] = future_replacement_cost
+    r["initial_capital_costs_after_incentives"] = r["lifecycle_capital_costs"] / p.third_party_factor - present_replacement_cost
+
+    r["replacements_future_cost"] = future_replacement_cost 
+    r["replacements_present_cost"] = present_replacement_cost 
     r["om_and_replacement_present_cost_after_tax"] = present_replacement_cost + r["lifecycle_om_costs_after_tax"]
     r["developer_om_and_replacement_present_cost_after_tax"] = r["om_and_replacement_present_cost_after_tax"] / 
         p.third_party_factor
 
     if !isempty(p.techs.fuel_burning)
         r["lifecycle_fuel_costs_after_tax"] = value(m[:TotalFuelCosts]) * (1 - p.s.financial.offtaker_tax_pct)
+    else
+        r["lifecycle_fuel_costs_after_tax"] = 0.0
+    end
+
+    # LCC breakdown:
+    r["lcc_fraction_tech_capital_cost"] = value(m[Symbol("TotalTechCapCosts"*_n)]) / r["lcc"] # Tech capital costs (including replacements)
```

Comment:
`lcc_..._cost` seems redundant. also, why do we need to report all these fractions? seems like a lot of redundant information as well. Why not just report the costs instead of dividing them by the lcc?